etna/datasets/tsdataset.py

```diff
@@ -1651,3 +1651,18 @@ def to_torch_dataset(
         ts_samples = [samples for df_segment in ts_segments for samples in make_samples(df_segment)]
 
         return _TorchDataset(ts_samples=ts_samples)
+
+    def size(self) -> Tuple[int, int, Optional[int]]:
+        """Return size of TSDataset.
+        Returns
+        -------
+        :
+            Number of time series, number of segments, and number of features (if their amounts are equal in each segment; otherwise, returns None)
```

Comment:
I think, we should write this info before "Returns", something like this:

```python
def size(self) -> Tuple[int, int, Optional[int]]:
        """Return size of TSDataset.

        The order of sizes is (number of time series, number of segments, 
        and number of features (if their amounts are equal in each segment; otherwise, returns None)).
        
        Returns
        -------
        :
            Tuple of TSDataset sizes 
        """
```